applications/OkAlice-Liquid-Staking.md

```diff
@@ -0,0 +1,153 @@
+- **Contact Email:** wouter@okalice.dev
+- **Website:** www.okalice.dev
+
+### Legal structure
+- In process.
+
+### Team's experience
+
+Our team's first project in the Polkadot ecosystem was 'Rooster Dao'. This project won 3 prizes during the Polkadot North-America hackathon in the summer of 2022:
+
+- Winner of the Community Choice award
+- First place for the RMRK challenges
+- Second place for Dao & Governance
+
+Based on the success of 'Rooster Dao', we were invited by the Astar team to adapt the contract for use on the Swanky node.
+
+In 2023, we participated in the Polkadot EU hackathon with a project called 'Pallet Rent'. This project was awarded the first prize in the NFT category.
+
+### Team Code Repos
+
+- https://github.com/ok-Alice
+- https://github.com/RoosterDao
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/kenneth-verbeure-2b151571/
+- https://www.linkedin.com/in/michael-assaf/
+- https://www.linkedin.com/in/woutergodefroy/
+- https://www.linkedin.com/in/dagmar-duportail-50919150/
+- https://www.linkedin.com/in/samuel-yi-54796b141/
+- https://www.linkedin.com/in/am-prajwal/
+
+
+
+## Development Status :open_book:
+
+## Development Roadmap :nut_and_bolt:
+### Overview 
+
+- **Total Estimated Duration:** 5 months
+- **Full-Time Equivalent (FTE):**  1.5
+- **Total Costs:** 55,000 USD
+
+### Milestone 1: Validator Selector
+
+- **Estimated Duration:** 2 months 
+- **FTE:**  1.5 
+- **Costs:** 22,000 USD 
+
+| Number | Deliverable | Specification |
+| ------:| ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | Inline documentation of the code and a basic tutorial on using the Substrate nodes and test transactions |
+| **0c.** | Testing and Testing Guide | Comprehensive unit tests and a guide on running these tests |
+| **0d.** | Docker | Dockerfile(s) for testing the milestone functionality |
+| 1. | XCM CE | Implementation of XCM chain-extension for nomination pool manipulation |
+| 2. | Oracle | Implementation of a simple oracle for publishing validator statistics |
+| 3. | Oracle prov | Backend scripts for provisioning oracle data |
+| 4. | Validator Selector | Contract implementation for optimized selection of nomination pool based on oracle's published statistics |
+
+### Milestone 2: Issuer-Staker
+
+- **Estimated Duration:** 2 months 
+- **FTE:**  1.5 
+- **Costs:** 22,000 USD 
+
+| Number | Deliverable | Specification |
+| ------:| ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | Inline documentation of the code and a basic tutorial on using the Substrate nodes and test transactions |
+| **0c.** | Testing and Testing Guide | Comprehensive unit tests and a guide on running these tests |
+| **0d.** | Docker | Dockerfile(s) for testing the milestone functionality |
+| 1. | Issuer-Staker | Contract implementation providing PSP22 interface for the derivative token and managing staking operations |
```

Comment:
Please consider how other projects might integrate with yours outside the context of an individual user with a wallet. What API might another contract require to build upon yours? (rhetorical question)

Reply:
Thank you for this suggestion. 
We are definitely  considering talking to the teams of NovaWallet, Interlay, Mantanetwork, Centrifuge, to hear if they might be interested in integrating our liquid derivatives or for NovaWallet, they might want to integrate the liquid staking option directly in their wallet?
We are eager to hear if you would have more suggestions?